capstone-ui/src/Components/App/App.jsx

```diff
@@ -20,20 +20,43 @@ export default function App() {
     localStorage.setItem("user", JSON.stringify(user));
   }, [user])
 
-
-
   const handleLogout = () => {
     updateUser(null);
   };
 
+  const [dropdownVal, setDropdownVal] = useState(window.location.pathname);
+
+  const options = [
```

Comment:
I think we probably can move these to a new file to avoid cluttering your `App.jsx` file. This way things can be more organized and won't have to worry about moving unrelated items around to make code clearer.